unit_tests/src/test_fixed_buffer_resource.cpp

```diff
@@ -63,7 +63,7 @@ TEST_CASE("fixed_buffer_resource throws on invalid arguments")
 
 TEST_CASE("fixed_buffer_resource works with pmr containers")
 {
-    char buffer[64];
+    char buffer[256];
```

Comment:
Seeing this solution makes me wonder if we should leave more words in the commit message or a comment here

Reply:
i'll add more comments to the commit message.

i believe we never saw this because this type was disabled until we introduced a version of boost that was greater than or equal to 1.72.

Reply:
done.